lib/middleware.js

```diff
@@ -48,23 +46,17 @@ module.exports = function middlewareConstructor(env) {
           ],
           'script-src': [
             "'self'",
```

Comment:
While you're in here, can you please add https://www.google-analytics.com to the list, it's failing on Thimble, so we'll need it.